extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -65,27 +58,30 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 		var gads = doc.createElement('script'),
 			node = doc.getElementsByTagName('script')[0];
 
-		gads.async = true;
-		gads.type = 'text/javascript';
-		gads.src = '//www.googletagservices.com/tag/js/gpt.js';
-		gads.addEventListener('load', function () {
-			log('GPT loaded', 'debug', logGroup);
+		window.googletag = window.googletag || {};
+		window.googletag.cmd = window.googletag.cmd || [];
+
+		if (!window.googletag.apiReady) {
+			gads.async = true;
+			gads.type = 'text/javascript';
+			gads.src = '//www.googletagservices.com/tag/js/gpt.js';
+
+			log('Appending GPT script to head', 'debug', logGroup);
+			node.parentNode.insertBefore(gads, node);
 			if (typeof onLoadCallback === 'function') {
 				onLoadCallback();
 			}
-		});
-
-		log('Appending GPT script to head', 'debug', logGroup);
-		node.parentNode.insertBefore(gads, node);
-
-		window.googletag = window.googletag || {};
-		window.googletag.cmd = window.googletag.cmd || [];
+		} else {
+			log('GPT script already appended', 'debug', logGroup);
+			onLoadCallback();
```

Comment:
Not sure if this callback is still needed. We may execute it in `gpt/helper.js` when recovery is enabled and blocking is detected.